ui/common/Tooltip.tsx

```diff
@@ -164,11 +164,15 @@ function Tooltip({
         aria-describedby={id}
       >
         {href && (
-          <a href={href} className={className}>
+          <a href={href} className={clsx('underline', className)}>
             {children}
           </a>
         )}
-        {!href && <span className={className}>{children}</span>}
+        {!href && (
+          <span className={clsx('border-b border-dotted', className)}>
```

Comment:
Thanks @benalleng , i was doing a decoration-dotted (so that didn't work for obvious reasons lol).
i have added a new commit to fixes this and also added a no-underline for the chapters navbar tooltip, because it got affected